pages/guides/developing/listen-for-webhooks.en-US.mdx

```diff
@@ -33,7 +33,7 @@ to a URL that you specify.
 | `multistream.error`        | This fires when we've encountered an error either while attempting to connect to the third party streaming service or while broadcasting.                              |
 | `multistream.disconnected` | This fires when we are no longer sending video to the multistream target.                                                                                              |
 | `asset.created`            | This fires when a On Demand asset is created.                                                                                                                          |
-| `asset.updated`            | This fires when a On Demand asset is updated.                                                                                                                          |
+| `asset.updated`            | This fires when a On Demand asset is updated. The asset payload will contain a playback URL when playback is ready.                                                    |
 | `asset.ready`              | This fires when a On Demand asset is ready.                                                                                                                            |
```

Comment:
I think a corresponding note here that playback will be "_ready_ with all transcoded renditions" or something like that would be useful as well, to contrast with the above. WDYT?

Reply:
yeah that sounds good